frontend/express/public/localization/dashboard/dashboard.properties

```diff
@@ -40,6 +40,7 @@ common.done = Done
 common.save = Save
 common.saving = Saving
 common.close = Close
+common.copy-id = Copy Id
```

Comment:
Where do we use this one?

Reply:
![screen shot 2018-07-04 at 14 19 27](https://user-images.githubusercontent.com/6316032/42274412-4284b336-7f95-11e8-9846-1b518dcb4ad5.png)